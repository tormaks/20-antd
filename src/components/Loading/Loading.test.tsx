import React from 'react';
import {render, screen} from '@testing-library/react';

import Loading from './index';

describe('Loading snapshot', () => {
  it ('AppHeader snapshot didnt change', () => {
    const app = render(<Loading/>)

    expect(app).toMatchSnapshot();
  });
});

describe('The Loading component', () => {
  it('should be render', () => {
    render(<Loading/>);
  })
})
