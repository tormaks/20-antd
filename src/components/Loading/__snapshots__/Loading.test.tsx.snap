// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot AppHeader snapshot didnt change 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="component"
      >
        <svg
          height="200px"
          preserveAspectRatio="xMidYMid"
          style="margin: 0px auto; background: none; display: block;"
          viewBox="0 0 100 100"
          width="200px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            transform="translate(80,50)"
          >
            <g
              transform="rotate(0)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="1"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="-0.875s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="-0.875s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
          <g
            transform="translate(71.21320343559643,71.21320343559643)"
          >
            <g
              transform="rotate(45)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="0.875"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="-0.75s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="-0.75s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
          <g
            transform="translate(50,80)"
          >
            <g
              transform="rotate(90)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="0.75"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="-0.625s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="-0.625s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
          <g
            transform="translate(28.786796564403577,71.21320343559643)"
          >
            <g
              transform="rotate(135)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="0.625"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="-0.5s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="-0.5s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
          <g
            transform="translate(20,50.00000000000001)"
          >
            <g
              transform="rotate(180)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="0.5"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="-0.375s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="-0.375s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
          <g
            transform="translate(28.78679656440357,28.786796564403577)"
          >
            <g
              transform="rotate(225)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="0.375"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="-0.25s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="-0.25s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
          <g
            transform="translate(49.99999999999999,20)"
          >
            <g
              transform="rotate(270)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="0.25"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="-0.125s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="-0.125s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
          <g
            transform="translate(71.21320343559643,28.78679656440357)"
          >
            <g
              transform="rotate(315)"
            >
              <circle
                cx="0"
                cy="0"
                fill="#FFFFFF"
                fill-opacity="0.125"
                r="7"
              >
                <animatetransform
                  attributeName="transform"
                  begin="0s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  type="scale"
                  values="1.5 1.5;1 1"
                />
                <animate
                  attributeName="fillOpacity"
                  begin="0s"
                  dur="1s"
                  keyTimes="0;1"
                  repeatCount="indefinite"
                  values="1;0"
                />
              </circle>
            </g>
          </g>
        </svg>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="component"
    >
      <svg
        height="200px"
        preserveAspectRatio="xMidYMid"
        style="margin: 0px auto; background: none; display: block;"
        viewBox="0 0 100 100"
        width="200px"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          transform="translate(80,50)"
        >
          <g
            transform="rotate(0)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="1"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="-0.875s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="-0.875s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
        <g
          transform="translate(71.21320343559643,71.21320343559643)"
        >
          <g
            transform="rotate(45)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="0.875"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="-0.75s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="-0.75s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
        <g
          transform="translate(50,80)"
        >
          <g
            transform="rotate(90)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="0.75"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="-0.625s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="-0.625s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
        <g
          transform="translate(28.786796564403577,71.21320343559643)"
        >
          <g
            transform="rotate(135)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="0.625"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="-0.5s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="-0.5s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
        <g
          transform="translate(20,50.00000000000001)"
        >
          <g
            transform="rotate(180)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="0.5"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="-0.375s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="-0.375s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
        <g
          transform="translate(28.78679656440357,28.786796564403577)"
        >
          <g
            transform="rotate(225)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="0.375"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="-0.25s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="-0.25s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
        <g
          transform="translate(49.99999999999999,20)"
        >
          <g
            transform="rotate(270)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="0.25"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="-0.125s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="-0.125s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
        <g
          transform="translate(71.21320343559643,28.78679656440357)"
        >
          <g
            transform="rotate(315)"
          >
            <circle
              cx="0"
              cy="0"
              fill="#FFFFFF"
              fill-opacity="0.125"
              r="7"
            >
              <animatetransform
                attributeName="transform"
                begin="0s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                type="scale"
                values="1.5 1.5;1 1"
              />
              <animate
                attributeName="fillOpacity"
                begin="0s"
                dur="1s"
                keyTimes="0;1"
                repeatCount="indefinite"
                values="1;0"
              />
            </circle>
          </g>
        </g>
      </svg>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
